presto-tests/src/main/java/io/prestosql/tests/AbstractTestQueryFramework.java

```diff
@@ -96,6 +101,7 @@ public void close()
         h2QueryRunner = null;
         sqlParser = null;
         queryRunnerSupplier = null;
+        LOG.info("Tests from %s took %s", getClass().getName(), Duration.nanosSince(started));
```

Comment:
If you do this as TestNG listener, this will be applicable to every test class, not only those inheriting from ATQF.

Reply:
Did you mean: https://testng.org/doc/documentation-main.html#listeners-service-loader? o_O

Reply:
I don't think we want to log for very test class. It's not valuable for most unit tests and will flood the logs.